packages/sdk/Dockerfile

```diff
@@ -169,5 +169,25 @@ ADD --chmod=644 \
 RUN tar -xJf /usr/share/cartesi-machine/images/linux-headers.tar.xz -C / && \
     rm /usr/share/cartesi-machine/images/linux-headers.tar.xz
 
+# Install migrate db cli
+ARG MIGRATE_VERSION=v4.18.2 
+RUN ARCH=$(dpkg --print-architecture || uname -m) && \
+    case "$ARCH" in \
```

Comment:
Can we follow the usage of `TARGETOS` and `TARGETARCH` that's already used in other places of this file?